vsts/windowsLinuxAndAndroidBuildMatrixConfig.yaml

```diff
@@ -241,7 +243,7 @@ jobs:
       - task: Bash@3
         condition: eq(variables['task.android.needToRunTestGroup'], 'yes')
         displayName: 'Start Android Emulator'
-        timeoutInMinutes: 10
```

Comment:
The emulator tasks have hit this 10 minute mark and failed even though the emulator is ready to use in the next step, so I've bumped up the timeout to allow for slower emulator start up times